src/extensions/contact-loaders/civicrm/const.js

```diff
@@ -0,0 +1,52 @@
+// This contains constants used by the CiviCRM contact loader and action
+// handlers. They are in their own file rather that others (like util.js),
+// This means that their value does not disappear when other JavaScript
+// files are mocked for testing.
+
+// Caching for action handlers and contact loaders should be a length of
+// a hour.
+
```

Comment:
are any of these constants something that should be customizable?  If so, at least in their use they should use the getConfig pattern

Reply:
Bumping this question for @johntwyman !

Reply:
Thanks @crayolakat! We have to update the PR more broadly anyway, so we'll make sure we consider this question as part of that process.

Reply:
Hi @crayolakat @schuyler1d - an update for you.

We've refreshed the code in this branch to ensure it has all the latest changes we made since raising the PR. We've also merged in upstream/main so the PR is aligned with the current codebase.

Re: the question @schuyler1d posed above about whether the constants in `src/extensions/contact-loaders/civicrm/const.js` are designed to be customisable, we're satisfied that for now they should _not_ be customisable, so the current code pattern is suitable.

We may look at making the cache duration and pagination size customisable in future, but usage of this code in production shows there's relatively little benefit in changing these values.

We're satisfied this PR is now ready for merging. We hope you are too. :)

Reply:
thanks @johntwyman !